File: Iniciante/1043/1043.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);

    String[] values = scan.nextLine().split(" ");

    float a = Float.parseFloat(values[0]);
    float b = Float.parseFloat(values[1]);
    float c = Float.parseFloat(values[2]);

    if (a < b + c && b < a + c && c < a + b) {
      System.out.printf("Perimetro = %.1f\n", a + b + c);
    } else {
      System.out.printf("Area = %.1f\n", (a + b) * c / 2);
    }
  }
}

File: Iniciante/1043/1043.py
# -*- coding: utf-8 -*-

values = input().split(" ")

a = float(values[0])
b = float(values[1])
c = float(values[2])

if a < b + c and b < a + c and c < a + b:
    print(f"Perimetro = {(a + b + c):.1f}")
else:
    print(f"Area = {(a + b) * c / 2:.1f}")
